ledger/participant-integration-api/src/main/scala/platform/DispatcherState.scala

```diff
@@ -24,53 +24,54 @@ class DispatcherState(dispatcherShutdownTimeout: Duration)(implicit
 ) {
   private val logger = ContextualizedLogger.get(getClass)
   @volatile private var dispatcherStateRef: DispatcherState.State =
-    DispatcherNotInitialized
+    DispatcherNotRunning
 
-  def initialized: Boolean = dispatcherStateRef match {
-    case _: DispatcherState.DispatcherOperational => true
-    case DispatcherState.DispatcherNotInitialized | DispatcherState.DispatcherShutdown => false
+  def getDispatcher: Dispatcher[Offset] = dispatcherStateRef match {
+    case DispatcherNotRunning => throw dispatcherNotRunning()
+    case DispatcherStateShutdown => throw dispatcherStateShutdown()
+    case DispatcherRunning(dispatcher) => dispatcher
   }
 
-  def getDispatcher: Dispatcher[Offset] = dispatcherStateRef match {
-    case DispatcherState.DispatcherNotInitialized => throw dispatcherNotInitializedException()
-    case DispatcherState.DispatcherOperational(dispatcher) => dispatcher
-    case DispatcherState.DispatcherShutdown => throw dispatcherShutdownException()
+  def startDispatcher(ledgerEnd: LedgerEnd): Unit = synchronized {
```

Comment:
It would be nice to add logging for starting the dispatcher as well, as we log other life-cycle events too.

Reply:
Will do.